Dockerfile

```
@@ -2,6 +2,7 @@ FROM python:3.7
 
 ADD . /gluonts
 
+RUN pip install mxnet==1.6
```

```suggestion
RUN pip install -r /gluonts/requirements/requirements-mxnet.txt
```

This should ensure that we install the currently recommended mxnet version.